taxcalc/functions.py

```diff
@@ -591,6 +575,22 @@ def TaxInc(c00100, c04470, c04100, _standard, e37717, c21060, c21040,
             _feitax, _oldfei)
 
 
+@iterate_jit(nopython=True, puf=False)
+def Personal_Credit(c04500, MARS, II_credit, II_credit_ps, II_credit_prt):
+    # full amount as defined in the parameter
+    _personal_credit = II_credit[MARS - 1]
+
+    # phaseout using AGI
```

Comment:
Also looks like the meta data for the parameters should reflect what the phaseout variable is.